### DISCUSSION TITLE
User guide to demonstrate the implementation of geotagging model

### DISCUSSION THREAD

**Discussion by AnuravModak** [1712n/yachay-public#16] [Q&A]
Is there any user guide available explaining how to implement and use the model.. something like this [click here](https://scikit-learn.org/stable/modules/generated/sklearn.neighbors.KNeighborsClassifier.html),( i have attached the example of user-guide for knn from scikit-learn.)

**Comment by ingakaspar** [comment]
@AnuravModak we just added a step by step demo for training https://github.com/Yachay-AI/byt5-geotagging#architecture-and-training let me know if that helps
